app/views/user_mailer/_request_expired.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Your <%= link_to 'reservation request', reservation_url(@reservation, only_path: false) %> has expired.<p>
+
+<p> You can request the equipment again <%= link_to 'here', equipment_model_url(@reservation.equipment_model) %>
```

Comment:
It's a shame we can't pass a link that creates a cart with the specified equipment and a specific start and due date. Hopefully that will happen when we API-ify all the things in #1061  :stuck_out_tongue:.